src/index-node.ts

```diff
@@ -28,10 +28,19 @@ export class SiweClient extends SiweImpl {
 
   async _extractCookies(headers?: Headers): Promise<void> {
     const cookieRecord: Record<string, string> = {}
-    const setCookie = Cookie.parse(headers?.get('set-cookie') || '')
-    if (setCookie) {
-      cookieRecord[setCookie.key] = setCookie.value
+    const headerSetCookies = (headers as any).raw()[
```

Comment:
Can we add a unit test for extractCookies? You can pass a mocked header object directly to _extractCookies instead of relying on `login` and fetch mock to mock headers